tests/test_envs.py

```diff
@@ -79,6 +79,10 @@ def test_routing(env_cls, batch_size=2, size=20):
 def test_mtvrp(variant, batch_size=2, size=20):
     env = MTVRPEnv(generator_params=dict(num_loc=size, variant_preset=variant))
     reward, td, actions = rollout(env, env.reset(batch_size=[batch_size]), random_policy)
+    try:
+        env.local_search(td, actions)
```

Comment:
No, I only added LS to TSP and CVRP.
I don't remember if I added the lines. This might be due to the unexpected behavior of git merge.
I will just remove it for now!

Reply:
> We should have this available for all 16 variants with the code @leonlan made

Where can I find the code? Is that about local search?

Reply:
It's about full solutions, but my guess is that it will work with any variant with minor modifications! [Here](https://anonymous.4open.science/r/routefinder/routefinder/baselines/pyvrp.py) is the code~